Firestore/core/src/firebase/firestore/model/document_key.h

```diff
@@ -90,6 +90,11 @@ class DocumentKey {
     return path_ ? *path_ : Empty().path();
   }
 
+  /** Returns true if the document is in the specified collectionId. */
+  bool HasCollectionID(const absl::string_view collectionID) const {
+    return path().size() >= 2 && path()[path().size() - 2] == collectionID;
```

Comment:
A couple of nits:
* remove `const` from the argument;
* s/collectionID/collection_id.

Also, perhaps storing size in an intermediate variable would make the expression slightly easier to parse?
```cpp
size_t size = path().size();
return size >= 2 && path()[size - 2] == collection_id;
```
Might not be worth backporting, though.

Reply:
Done.  FWIW- This is another case where I copied the const from elsewhere in this file (which I also fixed).